src/components/homePage.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FaCircle, FaSearch } from 'react-icons/fa';
+import { fetchCharacters } from '../redux/API/apiSlice';
+import './home.css';
+
+const HomePage = () => {
+  const [input, setInput] = useState('');
+  const dispatch = useDispatch();
+  const characters = useSelector((state) => state.api.data);
+  const status = useSelector((state) => state.api.status);
+  const error = useSelector((state) => state.api.error);
+  const [filteredCharacters, setFilteredCharacters] = useState(characters.results);
+
+  useEffect(() => {
+    dispatch(fetchCharacters());
+  }, [dispatch]);
+
+  useEffect(() => {
+    setFilteredCharacters(characters.results);
+  }, [characters.results]);
+
```

Comment:
- [ ] You did great on the project but you need to fix the below design issues to follow the given design.
  - Please make sure to display two items(star wars characters) in a row as on the design. Currently, there's one item displayed in a row.

### Your design

![image](https://github.com/vanina123/react-capstone/assets/83803414/49ed1ebd-c296-46bf-89cf-9e57bd1d888e)

### Given design

![image](https://github.com/vanina123/react-capstone/assets/83803414/893cc7dc-8567-412f-9e71-a3b1aacaf7e2)